### PR TITLE
Hide publish bar when typing on smaller devices

### LINKED ISSUE #435: The publish bar should disappear making more space in small screen.
When we open the editor on the small screen like mobile then the keyboard appears and at the top of the keyboard, there is publish bar. This publish bar makes the look weird and seems less spacing.

We should make the alignment of publish bar such that if we click on any input bar the publish bar disappears and when we click anywhere else the publish bar again appears.

editor

### PR DESCRIPTION
**Pull request by Shulammite-Aso**
Fixes #435 

pulled this out from here https://github.com/publiclab/PublicLab.Editor/pull/464 so it gets a little easier to review. The original PR had two changes in it

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt jasmine`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

File: src/core/Util.js [hunk 1]
```diff
@@ -126,6 +126,21 @@ module.exports = {
     var wysiwygDivObserver = new MutationObserver(handleChange);
 
     wysiwygDivObserver.observe(wysiwygDiv, observerConfig);
+  },
+
+  hideFooterWhenTypingOnMobile: function() {
+    var inputArea = $('input, textarea, .wk-wysiwyg');
+
+    if(window.innerWidth <= 992) {
```
Comment (sagarpreet-chadha) [inline src/core/Util.js:134]: I think this `if  condition` needs to be inside `focusin` listener?
For example, i am using a tablet and the browser width is less than 992. Now I have maximized the browser such that it width is greater than 992, in that case I donot want the bar to be hidden, right?

**Comment by Shulammite-Aso** [comment]
Oh just saw i have some unrelated changes here. will need to remove, just in case we'll be working with this PR.

**Comment by sagarpreet-chadha** [comment]
Correct, I generally ignore `/dist` file while reviewing

File: src/core/Util.js (hunk 1 above)
Comment (Shulammite-Aso) [reply src/core/Util.js:134]: will change this.

**Comment by Shulammite-Aso** [comment]
@sagarpreet-chadha please, i don't understand why this PR is showing a commit i made in the past "add bug report button", is that okay?  Also i don't know if that is why the tests are not passing?

please help. thanks!!

**Comment by sagarpreet-chadha** [comment]
Oh /dist has unrelated changes, you can do any of these things to remove  the first commit:
1. You can cherry pick the last 3 commits and add to a new branch forked from `main` branch.
2. As /dist file only has those changes, you can empty the content from that file and run the `grunt` command again to make that file.
3. Also as the change is small in this PR, you can make a new PR as well.

Thanks 😄

**Comment by Shulammite-Aso** [comment]
@sagarpreet-chadha i think something may be wrong with my fork. All the branches i create seem to already have some commits ahead, **on** my remote origin, and the commits are those from the past. But my local branches are okay, they don't have the extra commits.  I think i may have to delete my fork and and then fork again?

**Comment by sagarpreet-chadha** [comment]
Hey @Shulammite-Aso ,
I think you may have accidently commited on your local `main` branch of your forked repository, due to which any branck you fork has that extra commit maybe?
Can you try hard rebasing your `main` branch with origin `main` branch? Thanks 😄

**Pull request closed without merging**